documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+| Validated in lab/productive environments? | YES - app using APIs validated in CableLabs lab with potential field trials coming soon with network operators.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| Validated with real customers?            | NO - not yet, but CableLabs is currently in talks with both Meta and Zoom. Use case for both are for video conferencing applications to be able to monitor video, send KPIs, and trigger events if thresholds drop below certain levels.                                                                                                                                                                                                                                                                                                                                                            |
+| Validated with operators?                 | NO - but actively working with Charter, Liberly Global, and Vodaphone to get up and running in their labs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| Supporters in API Backlog Working Group   | Charter, Liberty Global, Vodaphone                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
How can Charter, Liberty Global, Vodafone (SP) support this if they haven't validated it?  This could be viewed as a red flag or at least cause for concern if we are saying that they support it, but are willing to say it's validated.

Reply:
udpated based on conversation with me, you, and Paul